src/scopes/activityReport/reportText.js

```diff
@@ -1,9 +1,58 @@
 import { Op } from 'sequelize';
+import { filterAssociation } from './utils';
 
-export function withReportText(reportIds) {
-  return { id: { [Op.in]: reportIds } };
+const nextSteps = `
+SELECT DISTINCT
+  "NextSteps"."activityReportId"
+FROM "NextSteps" "NextSteps"
+WHERE "NextSteps".note`;
+
+const args = `
+SELECT DISTINCT
+  "ActivityReportGoals"."activityReportId"
+FROM "ActivityReportGoals" "ActivityReportGoals"
+WHERE "ActivityReportGoals".name`;
+
+const objectiveTitle = `
+SELECT DISTINCT
+  "ActivityReportObjectives"."activityReportId"
+FROM "ActivityReportObjectives" "ActivityReportObjectives"
+WHERE "ActivityReportObjectives".title`;
+
+const objectiveTtaProvided = `
+SELECT DISTINCT
+  "ActivityReportObjectives"."activityReportId"
+FROM "ActivityReportObjectives" "ActivityReportObjectives"
+WHERE "ActivityReportObjectives"."ttaProvided"`;
+
+const goalResources = `
+SELECT DISTINCT
+  "ActivityReportObjectiveResources"."activityReportId"
+FROM "ActivityReportObjectiveResources" "ActivityReportObjectiveResources"
+INNER JOIN "Resources" "Resources"
+ON "Resources"."id" = "ActivityReportObjectiveResources"."resourceId"
+WHERE "Resources"."url"`;
```

Comment:
This should be done for ActivityReportResource, ActivityReportGoalResource, ActivityReportObjectiveResource, NextStepsResource